bld/unit_testers/build-namelist_test.pl

```diff
@@ -1061,6 +1061,11 @@ sub cat_and_create_namelistinfile {
                                      GLC_TWO_WAY_COUPLING=>"FALSE",
                                      phys=>"clm5_0",
                                    },
+     "dogrossandsp"               =>{ options=>"-envxml_dir . -bgc sp -use_case 20thC_transient",
```

Comment:
We could add two more fail tests here. One for turning do_grossunrep on for a non transient case. Or for a transient FATES case.

Reply:
I went ahead and added these tests in and this is good.